File: api/models/message.py
from app import db
from user import User


class Message(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    cre_dt = db.Column(db.DateTime)
    transaction_id = db.Column(db.Integer, db.ForeignKey('transaction.id'))
    to_user_id = db.Column(db.Integer, db.ForeignKey(User.id))
    to_user = db.relationship(User, uselist=False, foreign_keys=[to_user_id])
    from_user_id = db.Column(db.Integer, db.ForeignKey(User.id))
    from_user = db.relationship(User, uselist=False, foreign_keys=[from_user_id])
    content = db.Column(db.String(1000))

File: mobile/app/models/message.ts
import {UserPublic} from './user_public'

export class Message {
    
    public id:number;
    public to_user:UserPublic;
    public from_user:UserPublic;
    public content:string;
        
    constructor() {}
   
    public static fromJSON(jsonObj): Message {
        var message:Message = new Message();
        message.id = jsonObj.id;
        message.to_user = UserPublic.fromJSON(jsonObj.to_user);
        message.from_user = UserPublic.fromJSON(jsonObj.from_user);
        message.content = jsonObj.content;
        return message
    }
}